### PR TITLE
building authentication cookie headers in rake (for load testing scripts)

### PR DESCRIPTION
**Pull request by saneshark**
## Description of change
Extracts the session creation rake tasks from the `redis` namespace to `load_test`and updates them to output cookie headers that can be used in the updated load tests.

## Testing done
specs, ran load tests against a local rails instance

## Testing planned
staging load tests

#### Unique to this PR
- [ ] Updates the sessions rake tasks to use cookie based sessions rather than tokens
- [ ] Creates sessions allow load tests to run again

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by kreek** [comment]
@saneshark I think is ready to be tested on staging, related devops PR https://github.com/department-of-veterans-affairs/devops/pull/4021

File: rakelib/support/sessions_arg_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        }
+      }
+    }
+  end
+  # rubocop:enable Metrics/MethodLength
+
+  def identity_data(uuid)
+    {
+      ":uuid": uuid,
+      ":email": "vets.gov.user+#{rand(200)}@gmail.com",
```
Comment (kfrz) [inline rakelib/support/sessions_arg_serializer.rb]: Will this cause collision based failures on any given test run?

File: rakelib/support/sessions_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+require './rakelib/support/cookies.rb'
+
+class SessionsSerializer
+  def initialize(_arg)
+    @sessions = []
+    @redis = Redis.current
+  end
+
+  def load
+    raise 'subclass must implement #load'
```
Comment (kfrz) [inline rakelib/support/sessions_serializer.rb]: Is `load` part of a pattern that we're just not utilizing here? Neither of the subclasses implement nor use it from what I can see

Comment (kreek) [reply rakelib/support/sessions_serializer.rb]: good catch I moved that logic into each of the sub classes initialize method, will remove

File: rakelib/support/sessions_arg_serializer.rb (hunk 1 above)
Comment (kreek) [reply rakelib/support/sessions_arg_serializer.rb]: 🤔not sure, old tests had the same but code but maybe never ran into it, I'll update it so it's unique

**Pull request merged**